tensorflow/python/compiler/tensorrt/trt_convert_test.py

```diff
@@ -702,6 +702,44 @@ def _InputFn():
     del root_with_trt
     gc.collect()  # Force GC to destroy the TRT engine cache.
 
+  @test_util.run_v2_only
+  def testTrtGraphConverter_RemoveNativeSegments(self):
+    """Test case for trt_convert._remove_native_segment()."""
+    np_input = np.random.random_sample([5, 3]).astype(np.float32)
+    # Create a model and save it.
+    input_saved_model_dir = tempfile.mkdtemp(dir=self.get_temp_dir())
+    root = self._GetShapeOpModel()
+    expected_output = root.run(np_input)
+    save.save(root, input_saved_model_dir, signatures=root.run)
+
+    # Run TRT conversion.
+    converter = trt_convert.TrtGraphConverterV2(
+        input_saved_model_dir,
+        precision_mode=trt_convert.TrtPrecisionMode.FP32,
+        allow_build_at_runtime=False,
+        minimum_segment_size=1)
+
+    def _input_fn():
+      yield (np_input,)
+    graph_func = converter.convert()
+    converter.build(_input_fn)
+    # Load and verify the reduced converted model.
+    output_saved_model_dir2 = self.mkdtemp()
+    os.environ["TF_TRT_EXPERIMENTAL_FEATURES"] = "remove_native_segments"
+    converter.save(output_saved_model_dir2)
```

Comment:
We have a context manager for this
```suggestion
    with test_utils.experimental_feature_scope("remove_native_segments"):
      converter.save(output_saved_model_dir2)
```